src/views/AgmExtension.vue

```diff
@@ -277,8 +277,8 @@ export default class AgmExtension extends Mixins(CommonMixin, DateMixin, FilingM
     this.filingId = +this.$route.query.filingId // number or NaN
 
     // if required data isn't set, go back to dashboard
```

Comment:
This comment is now obsolete. See comment on line 276 and maybe move it here.

Reply:
Updated the comments